src/dsp_tools/commands/excel2json/models/json_header.py

```diff
@@ -0,0 +1,135 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any
+from typing import Protocol
+
+SCHEMA = "https://raw.githubusercontent.com/dasch-swiss/dsp-tools/main/src/dsp_tools/resources/schema/project.json"
+
+
+@dataclass
+class JsonHeader(Protocol):
+    def make(self) -> dict[str, Any]:
+        raise NotImplementedError
+
+
+@dataclass
+class EmptyJsonHeader(JsonHeader):
+    def make(self) -> dict[str, Any]:
+        return {
+            "prefixes": {"": ""},
+            "$schema": SCHEMA,
+            "project": {
+                "shortcode": "",
+                "shortname": "",
+                "longname": "",
+                "descriptions": {"en": ""},
+                "keywords": [""],
+            },
+        }
+
+
+@dataclass
+class ExcelJsonHeader(JsonHeader):
```

Comment:
Why Excel?

Reply:
Because the information comes from an excel file